Stages.md

```diff
@@ -0,0 +1,125 @@
+* _Browser engine representatives_: persons delegated by a browser engine to indicate implementation interest.
+* _Proposal champion_: person/organization that proposed and will advance a feature through the stages, typically including writing the specification, researching use cases, and prototyping.
+
+Process:
+
+* Any proposal effectively starts at Stage 0 without any approvals, by a community member filing a new issue on a relevant WHATWG spec.
+* In order to drive a proposal forward, it is expected that a community member will step up to champion the proposal, through answering questions asynchronously, providing feedback when requested, and consideration of important open questions at triage meetings.
+* Stage level is tracked with labels on issues in the relevant specifications — “stage 0”, “stage 1”, etc.
+* Stage labels (other than stage 0) should only be added by browser engine representatives, or by Editors on the relevant specification, after showing the necessary support for a level.
+* **Advancing stages**
+    * The feature champion should bring the proposal to the WHATWG to advance to the next stage by setting an “Agenda+” label on the tracking issue, and showing up to (or ensuring someone will show up to) the next triage meeting to discuss. Advancing to a new stage requires support for the decision by at least two implementers (via their browser engine representatives), and there should not be any strong implementer objections ([per the working mode](https://whatwg.org/working-mode#additions)). This support can also be gathered in any public manner, e.g. GitHub issue comments, triage meetings, etc. 
+    * Positive support to advance a stage does **not** imply:
+        * Commitment to eventually support a subsequent stage
+        * That the particular solution approach at the time of the signal or support will be the one eventually standardized (unless the advancement is to the Standardized stage)
+    * A negative response to advancing a stage does **not** imply:
+        * That the feature cannot continue in its current stage
+        * That a future request for support to advance a stage with a revised solution won’t succeed
+        * That all browsers will delay shipping the feature
+        * That the feature cannot continue to be incubated outside of the WHATWG
+    * A proposal can advance directly to later stages without going through the earlier stages, if it's already mature enough and has sufficient support to do so.
+* **Regressing stages**
+    * The browser engine representatives may review proposals and decide to regress stages—for example, if implementer support has been retracted for a particular solution (regressing from stage 3 to stage 2), or even if support has been retracted for solving the problem (i.e. a regression from stage 2 to stage 1). This situation is expected to be unusual, and should come with a very strong justification.
+
+<table>
+  <tr>
+   <td>
```

Comment:
Make these th instead of td, and probably it'd be good to introduce thead and tbodys.

Reply:
Done in https://github.com/whatwg/sg/pull/222/commits/dd195992df4f01dbd374454f6642065ec8a2bbe9.